scripts/run_settlement_probability_prd_gate.py

```
@@ -0,0 +1,321 @@
+        poll_seconds=args.poll_seconds,
+    )
+    if snapshot_result.conclusion != "success":
+        issue_comment(
+            args.issue_number,
+            "\n".join(
+                [
+                    "Settlement probability PRD gate blocked at strict snapshot:",
+                    "",
+                    f"- Snapshot run: {snapshot_result.url}",
+                    f"- Conclusion: `{snapshot_result.conclusion}`",
+                    f"- Data profile: `pm5d-vol`",
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)